### DISCUSSION TITLE
What causes "Last Login" to be older than "Last Seen"

### DISCUSSION THREAD

**Discussion by AdeonWriter** [vrcx-team/VRCX#336] [Q&A] [answered]
I have a few friends who have a "Last Seen" date of yesterday/today, but a "Last Login" date of some time last year.

I'm guessing these dates don't mean what I think they mean?

**Comment by SalbugVR** [comment, marked as answer]
Last login means when the API last detected them inputting their creds and logging in. Last seen just means the last time you came across them in an instance.
